examples/map/difference_images.py

```diff
@@ -18,41 +15,69 @@
 import sunpy.map
 
 ###########################################################################
-# First load a couple of images taken from the sample dataset. These are
-# two cutouts taken during a flare.
-maps = sunpy.map.Map([sunpy.data.sample.AIA_193_CUTOUT03_IMAGE,
-                      sunpy.data.sample.AIA_193_CUTOUT04_IMAGE],
-                     sequence=True)
+# When analyzing solar imaging data, particularly for flares, it is often
+# useful to look at the difference from one time step to the next (running
+# difference) or the difference from the start of the sequence (base
+# difference). In this example, we'll use a sequence of AIA 193 cutout
+# images taken during a flare.
+#
+# First, load a series of images into a `~sunpy.map.MapSequence`.
 
-###########################################################################
-# Now we'll do a standard plot of the second image just to see it.
-
-plt.figure()
-ax = plt.subplot(projection=maps[1])
-maps[1].plot(clip_interval=(0.5, 99.9)*u.percent)
+m_seq = sunpy.map.Map([
+    sunpy.data.sample.AIA_193_CUTOUT01_IMAGE,
+    sunpy.data.sample.AIA_193_CUTOUT02_IMAGE,
+    sunpy.data.sample.AIA_193_CUTOUT03_IMAGE,
+    sunpy.data.sample.AIA_193_CUTOUT04_IMAGE,
+    sunpy.data.sample.AIA_193_CUTOUT05_IMAGE,
+], sequence=True)
 
 ###########################################################################
-# And now we can do take the actual difference.
+# Let's take a look at each image in the sequence. Note that these images
+# are sampled at a 6.4 minute cadence, much lower than the actual 12 s
+# AIA cadence.
 
-diff = maps[1].data - maps[0].data
+fig = plt.figure(figsize=(12, 3))
+for i, m in enumerate(m_seq):
+    ax = fig.add_subplot(1, 5, i+1, projection=m)
+    delta_t = (m.date - m_seq[0].date).to('minute')
+    m.plot(axes=ax, clip_interval=(0.5, 99.9)*u.percent,
+           title=f'{delta_t:.2f}')
+    ax.coords[0].set_axislabel('Solar-X' if i == 0 else ' ')
+    ax.coords[0].set_ticklabel_visible(i == 0)
+    ax.coords[1].set_axislabel('Solar-Y' if i == 0 else ' ')
+    ax.coords[1].set_ticklabel_visible(i == 0)
+plt.show()
 
 ###########################################################################
+# And now we can take the actual difference. We will compute the running
+# difference and the base difference for the last image in the sequence.
+#
 # But we have to decide what to do with the metadata. For example, what
 # time does this difference image correspond to? The time of the first or
 # second image? The mean time? You'll have to decide what makes most sense
-# for your application. Here we'll just use the metadata from the second
-# image. Then we can store the difference and header back in a Map.
+# for your application. Here, by subtracting the data of the first (and
+# second to last) data array from the last map in the sequence, the resulting
+# difference map has the same metadata as the last map in the sequence.
+
+m_diff_base = m_seq[-1] - m_seq[0].quantity
```

Comment:
This suggestion shouldn't work because we expressly do not support subtracting two maps.  One of the operands needs to be no more complex than a `Quantity`.

Reply:
Certainly the reason for converting one of the two operands should be explained in a content

Reply:
Yes, Albert is correct. We do not currently support arithmetic operations between two maps. By converting the second to a `Quantity` we're performing a (unit-aware) subtraction of the first array of the sequence from the last array and the resulting map inherits the metadata of the last array. I will add a comment that clarifies this.

Reply:
I've added a clarifying comment. I'm also now calculating the difference image for each map in the sequence.